alfresco-search/src/test/java/org/alfresco/solr/tracker/DistributedDateMonthAlfrescoSolrTrackerTest.java

```diff
@@ -130,10 +132,21 @@ public void testDateMonth() throws Exception
         AlfrescoSolrDataModel.FieldInstance fieldInstance = fieldInstanceList.get(0);
         String fieldName = fieldInstance.getField();
 
-        for (int i = 0; i < dates.length; i++) {
-            LegacyNumericRangeQuery query = LegacyNumericRangeQuery.newLongRange(fieldName, dates[i].getTime(), dates[i].getTime() + 1, true, false);
-            assertCountAndColocation(query, counts[i]);
-            assertShardSequence(i, query, counts[i]);
+
+        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+        format.setTimeZone(TimeZone.getTimeZone("GMT"));
+        for (int i = 0; i < dates.length; i++)
+        {
+            String startDate = format.format(dates[i]);
+            Calendar gcal = new GregorianCalendar();
+            gcal.setTime(dates[i]);
+            gcal.add(Calendar.SECOND, 1);
+            String endDate = format.format(gcal.getTime());
+
+            SolrQuery solrQuery = new SolrQuery("{!lucene}" + escapeQueryChars(fieldName) +
+                    ":[" + escapeQueryChars(startDate) + " TO " + escapeQueryChars(endDate) + " } " );
+            assertCountAndColocation(solrQuery, counts[i]);
+            assertShardSequence(i, solrQuery, counts[i]);
```

Comment:
I think this would benefit from a comment to say what this block of code does. The test was long and without comments before your refactoring, but given you've been through it to work out what it does, then it would be good to document that.

Reply:
Ok - no problem :+1: